src/ec/packages/ec-component-button/button.story.js

```diff
@@ -57,6 +57,7 @@ storiesOf('Components/Button', module)
           type: 'ui',
           name: iconsListSelect,
           path: defaultSprite,
+          size: 'fluid',
```

Comment:
they are cases where the iconsize is larger than the text, for instance the icon before "Select your language" in the overlay language list

Reply:
"Select your language" is not a button. I don't follow you...

Reply:
My bad, this is the "close" button" on the same coponent

Reply:
the specs don't say anything about the close icon (nor about the language icon)... indeed it seems bigger than the text, but how much bigger?

could we use "fluid" as a default, unless specified otherwise?

Reply:
it has to be decided on ECL level.
Personally I would leave it as is unless we have a request to change it